cmd/argo/commands/get.go

```diff
@@ -443,8 +446,21 @@ func renderChild(w *tabwriter.Writer, wf *wfv1.Workflow, nInfo renderNode, depth
 
 // Main method to print information of node in get
 func printNode(w *tabwriter.Writer, node wfv1.NodeStatus, nodePrefix string, getArgs getFlags) {
-	if getArgs.status != "" && string(node.Phase) != getArgs.status {
-		return
+	if getArgs.status != "" {
```

Comment:
can we move this code into a single func

```
func (a getArgs) GetNodeFieldSelector() (fields.Selector, error)
```

Then you can unit test that function easily.

Reply:
Sure thing